step-controller/step-controller-remote-client/src/main/java/step/client/functions/RemoteFunctionExecutionService.java

```diff
@@ -102,4 +103,14 @@ public Type getOwnerType() {
 		
 		return executeRequest(()->b.post(entity,genericType));
 	}
+
+	@Override
+	public void registerTokenLifecycleInterceptor(TokenLifecycleInterceptor interceptor) {
+		// FIXME: does this need an implementation?

```

Comment:
@cl-exense this class is used by the step client, it will call the related REST service to reserve a token. The REST service  invokes the FunctionExecutionServiceImpl in the BE to actual get a token, so no implementation is required on the Remote class from my point of view.